CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Hardening: Several changes in argument passing in mongoBackend library to avoid passing entire objects on the stack, from "X x" to "const X& x"
 - Hardening: Several changes in argument passing in mongoBackend library to add 'const' in references to objects that are not altered by the function
 - Fix: bug in parseArg lib that may cause problem printing the error message for wrong CLI usage (#2926)
+- Fix: bug in variable substitution of custom notifications that limited the size of a custom notification to 1024 bytes
```

Comment:
"Fix: bug in variable substitution of custom notifications that limited the size of a custom notification payload to 1024 bytes (new limit: 8MB)"

(An of course the change to implement that new limit of 8MB ;))

Note the "payload" word. As far as I understand the bug was limiting the payload size, not the notification size (including headers, etc.).